components/Cards/DetailsCard.js

```diff
@@ -35,19 +47,135 @@ export default function DetailsCard({ id, isFavorite, toggleFavorite, place }) {
 
   const [comments, setComments] = useState([]);
   const [editingCommentId, setEditingCommentId] = useState(null);
+  const [editingPlaceId, setEditingPlaceId] = useState(false);
+  const [editingActivities, setEditingActivities] = useState([]);
+  const [editingImage, setEditingImage] = useState();
+  const [editingPublicId, setEditingPublicId] = useState();
```

Comment:
I know this is something you want to implement at a later point, so for this PR, you can remove those two states.
Or do you already use them someplace and I overlooked it?